### PR TITLE
Documentation about GPU

### PR DESCRIPTION
**Pull request by Jeronymous**
See new documentation: https://github.com/linto-ai/linto-stt/blob/wip/whisper/README.md

Also support smore model formats

### CODE REVIEW

File: whisper/Dockerfile.ctranslate2 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM ghcr.io/opennmt/ctranslate2:latest-ubuntu20.04-cuda11.2
-LABEL maintainer="jlouradour@linagora.com"
+LABEL maintainer="hello@linto.ai, jlouradour@linagora.com, dgaynullin@linagora.com"
```
Comment (Jeronymous) [inline whisper/Dockerfile.ctranslate2:2]: @damienlaine Should we use support@linto.ai instead of hello@linto.ai here?

Comment (damienlaine) [reply whisper/Dockerfile.ctranslate2:2]: Not sure if support is routed correctly, 
contact@linto.ai is the best

**Review by damienlaine — APPROVED** [review]
Seems okay.
We shall remove all references to "LinTO-Platform stack" as it references former smart assistant stack of services.
@gaydmi @wghezaiel @Siliam : same goes for every other repos (NLP...)

**Comment by Jeronymous** [comment]
> Seems okay. We shall remove all references to "LinTO-Platform stack" as it references former smart assistant stack of services. @gaydmi @wghezaiel @Siliam : same goes for every other repos (NLP...)

I removed those here : https://github.com/linto-ai/linto-stt/pull/36/commits/8410b34229431f0a8a87867842c541477ce2af04

**Pull request merged**